MapboxNavigation/Locale.swift

```diff
@@ -0,0 +1,15 @@
+import Foundation
+
+
+extension Locale {
+    
+    // Returns Locale.autoupdatingCurrent unless it is prefixed with `en`, then nil will be returned.
+    static var autoupdatingCurrentOrNil: Locale? {
+        if let languageCode = Locale.autoupdatingCurrent.languageCode,
+            languageCode.hasPrefix("en"){
```

Comment:
English isn’t the only ISO&nbsp;639 code that begins with `en`. Check for equality with `en` instead; Locale will never come up with an alpha-3 language code of `eng`.